src/commons/resources.rs

```diff
@@ -158,10 +158,14 @@ pub struct Resources<T, K = NoRuntimeLimits> {
     schemars(bound = "T: JsonSchema, T::Fragment: JsonSchema + Default")
 )]
 pub struct MemoryLimits<T> {
-    // The maximum amount of memory that should be available
-    // Should in most cases be mapped to resources.limits.memory
+    /// The maximum amount of memory that should be available to the Pod.
+    /// Specified as a byte [Quantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/),
+    /// which means these suffixes are supported: E, P, T, G, M, k.
+    /// You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki.
+    /// For example, the following represent roughly the same value:
+    /// `128974848, 129e6, 129M,  128974848000m, 123Mi`
     pub limit: Option<Quantity>,
-    // Additional options that may be required
+    /// Additional options that may be required.
```

Comment:
` #[fragment_attrs(serde(default))]` imposes the stuff has a default
```suggestion
    /// Additional options that can be specified.
```